tests/phpunit/tests/test-wpml-config-locations.php

```diff
@@ -96,14 +96,26 @@ public function test_in_mu_plugin() {
 
 		@symlink( PLL_TEST_DATA_DIR . 'plugins/best-plugin', WPMU_PLUGIN_DIR . '/best-plugin' );
 
-		$files    = ( new PLL_WPML_Config() )->get_files();
-		$expected = array(
-			'mu-plugins'             => $filename_1,
-			'mu-plugins/must-use'    => $filename_2,
-			'mu-plugins/best-plugin' => WPMU_PLUGIN_DIR . '/best-plugin/wpml-config.xml',
+		$files    = array_map( 'wp_normalize_path', ( new PLL_WPML_Config() )->get_files() );
+		$expected = array_map(
+			'wp_normalize_path',
+			array(
+				'mu-plugins'             => $filename_1,
+				'mu-plugins/must-use'    => $filename_2,
+				'mu-plugins/best-plugin' => WPMU_PLUGIN_DIR . '/best-plugin/wpml-config.xml',
+			)
 		);
 
-		unlink( WPMU_PLUGIN_DIR . '/best-plugin' );
+		if ( stripos( PHP_OS, 'WIN' ) === 0 ) {
+			/*
+			 * Uses rmdir() to remove symbolic link on Windows. See https://www.php.net/manual/fr/function.unlink.php
+			 * Because symbolic link could not be created before if we don't have permissions, it needs to protect rmdir() call to prevent any error.
```

Comment:
```suggestion
			 * Use rmdir() to remove symbolic link on Windows. See https://www.php.net/manual/en/function.unlink.php
			 * In case the symbolic link could not be created if we don't have permissions, we need to remove errors.
```